src/controllers/workspace.controller.js

```diff
@@ -78,9 +66,9 @@ exports.updateWorkspace = async (req, res) => {
         });
 
         return response_200(res, "Workspace Updated Successfully", {
-            name: workspace.name,
-            description: workspace.description,
-            workspaceId: workspace.workspaceId,
+            name: name || workspace.name,
+            description: description || workspace.description,
+            workspaceId: workspaceId || workspace.workspaceId,
```

Comment:
Why this is necessary??

Reply:
for return response, suppose admin is not requesting to change the name, then name will be null, so || workspace.name makes sure that if name is null, then previous name is returned